corehq/apps/cleanup/tasks.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
This file is called "tasks.py" because my plan is to add a periodic celery task that will run the command and email the summary statistics to relevant people.